src/euphorie/client/model.py

```diff
@@ -1621,6 +1621,33 @@ class OrganisationMembership(BaseObject):
     member_role = schema.Column(types.UnicodeText())
 
 
+class NotificationSubscription(BaseObject):
+    __tablename__ = "notification_subscription"
+
+    id = schema.Column(types.Integer(), primary_key=True, autoincrement=True)
+    account_id = schema.Column(
+        types.Integer(),
+        schema.ForeignKey(Account.id, onupdate="CASCADE", ondelete="CASCADE"),
+        nullable=False,
+    )
+    category = schema.Column(types.String(512), nullable=False)
+    enabled = schema.Column(types.Boolean(), nullable=False, default=False)
+
+
+class NotificationsSent(BaseObject):
```

Comment:
This is unused, right? Should we take it out for now?

Reply:
removed.